tests/apps/demultiplex/conftest.py

```diff
@@ -157,3 +158,164 @@ def fixture_novaseq_dragen_sample_sheet_object(
+    )
+
+
+@pytest.fixture(name="sample_sheet_bcl2fastq_duplicate_same_lane")
+def fixture_sample_sheet_bcl2fastq_duplicate_same_lane(valid_sample_sheet_bcl2fastq: str):
+    """Return the content of a bcl2fastq sample sheet with a duplicated sample in the same lane."""
+    return (
+        valid_sample_sheet_bcl2fastq
+        + "\n"
+        + "HWHMWDMXX,1,ACC7628A1,hg19,AGTTAGCTGG,GATGAGAATG,814206,N,R1,script,814206"
+    )
+
+
+@pytest.fixture(name="sample_sheet_bcl2fastq_duplicate_different_lane")
+def fixture_sample_sheet_bcl2fastq_duplicate_different_lane(valid_sample_sheet_bcl2fastq: str):
+    """Return the content of a bcl2fastq sample sheet with a duplicated sample in a different lane."""
+    return (
+        valid_sample_sheet_bcl2fastq
+        + "\n"
+        + "HWHMWDMXX,2,ACC7628A1,hg19,AGTTAGCTGG,GATGAGAATG,814206,N,R1,script,814206"
+    )
+
+
+@pytest.fixture(name="valid_sample_sheet_dragen")
+def fixture_valid_sample_sheet_dragen() -> str:
+    """Return the content of a valid dragen sample sheet."""
+    return (
+        "[Data]\n"
+        "FCID,Lane,Sample_ID,SampleRef,index,index2,SampleName,Control,Recipe,Operator,"
+        "Sample_Project\n"
+        "HWHMWDMXX,1,ACC7628A68,hg19,ATTCCACACT,TGGTCTTGTT,814206,N,R1,script,814206\n"
+        "HWHMWDMXX,1,ACC7628A1,hg19,AGTTAGCTGG,GATGAGAATG,814206,N,R1,script,814206"
+    )
+
+
+@pytest.fixture(name="sample_sheet_dragen_duplicate_same_lane")
+def fixture_sample_sheet_dragen_duplicate_same_lane(valid_sample_sheet_dragen: str):
+    """Return the content of a dragen sample sheet with a duplicated sample in the same lane."""
+    return (
+        valid_sample_sheet_dragen
+        + "\n"
+        + "HWHMWDMXX,1,ACC7628A1,hg19,AGTTAGCTGG,GATGAGAATG,814206,N,R1,script,814206"
+    )
+
+
+@pytest.fixture(name="sample_sheet_dragen_duplicate_different_lane")
+def fixture_sample_sheet_dragen_duplicate_different_lane(valid_sample_sheet_dragen: str):
+    """Return the content of aa dragen sample sheet with a duplicated sample in a different lane."""
+    return (
+        valid_sample_sheet_dragen
+        + "\n"
+        + "HWHMWDMXX,2,ACC7628A1,hg19,AGTTAGCTGG,GATGAGAATG,814206,N,R1,script,814206"
+    )
+
+
+@pytest.fixture(name="s2_sample_sheet_type")
+def fixture_s2_sample_sheet_type() -> str:
+    """Returns the S2 sample sheet type."""
+    return "S2"
+
+
+@pytest.fixture(name="bcl2fastq_converter")
+def fixture_bcl2fastq_converter() -> str:
+    """Return the name of the bcl2fastq converter."""
+    return "bcl2fastq"
+
+
+@pytest.fixture(name="dragen_converter")
+def fixture_dragen_converter() -> str:
+    """Return the name of the dragen converter."""
+    return "dragen"
+
+
+@pytest.fixture(name="valid_sample_sheet_bcl2fastq_path")
+def fixture_valid_sample_sheet_bcl2fastq_path() -> Path:
+    """Return the path to a NovaSeq S2 sample sheet, used in bcl2fastq demultiplexing."""
+    return Path("tests/fixtures/apps/demultiplexing/SampleSheetS2_Bcl2Fastq.csv")
```

Comment:
```suggestion
    return Path("tests", "fixtures", "apps", "demultiplexing", "SampleSheetS2_Bcl2Fastq.csv")
```
Avoid hard-coding "/"